src/data/inchi-dictionary.ts

```diff
@@ -2,430 +2,825 @@ import { INCHIProps } from '../interfaces/utils';
     molecularWeight: 101.96,
   },
+
   AluminumPhosphate: {
+    CAS: '7784-30-7',
     inchiString: 'InChI=1S/Al.H3O4P/c;1-5(2,3)4/h;(H3,1,2,3,4)/q+3;/p-3',
     inchiKey: 'ILRRQNADMUWWFW-UHFFFAOYSA-K',
     molecularWeight: 121.95,
   },
+
   AluminumPotassiumSilicateMuscovite: {
+    CAS: '',
     inchiString: '',
     inchiKey: '',
     molecularWeight: 796.61,
   },
+
   AluminumPotassiumSilicateOrthoclase: {
+    CAS: '',
     inchiString: '',
     inchiKey: '',
     molecularWeight: 556.66,
   },
+
   AluminumPotassiumTartrate: {
+    CAS: '',
     inchiString: '',
     inchiKey: '',
     molecularWeight: 362.22,
   },
+
   AluminumSodiumFluorideCryolite: {
+    CAS: '13775-53-6',
     inchiString: 'InChI=1S/Al.6FH.3Na/h;6*1H;;;/q+3;;;;;;;3*+1/p-6',
     inchiKey: 'REHXRBDMVPYGJX-UHFFFAOYSA-H',
     molecularWeight: 209.94,
   },
+
   AluminumSodiumSilicate: {
+    CAS: '',
     inchiString: '',
     inchiKey: '',
     molecularWeight: 524.44,
   },
+
   AluminumSulfate: {
+    CAS: '10043-01-3',
     inchiString:
       'InChI=1S/2Al.3H2O4S/c;;3*1-5(2,3)4/h;;3*(H2,1,2,3,4)/q2*+3;;;/p-6',
     inchiKey: 'DIZPMCHEQGEION-UHFFFAOYSA-H',
     molecularWeight: 342.15,
   },
+
   AmmoniumAlumTschermigite: {
+    CAS: '',
     inchiString: '',
     inchiKey: '',
     molecularWeight: 906.66,
   },
+
   AmmoniumChromeAlum: {
+    CAS: '',
     inchiString: '',
     inchiKey: '',
     molecularWeight: 956.69,
   },
+
   AmmoniumIronAlum: {
+    CAS: '',
     inchiString: '',
     inchiKey: '',
     molecularWeight: 964.38,
   },
+
   PotassiumAlumKaliniteAlum: {
+    CAS: '',
     inchiString: '',
     inchiKey: '',
     molecularWeight: 948.78,
   },
+
   PotassiumChromeAlum: {
+    CAS: '',
     inchiString: '',
     inchiKey: '',
     molecularWeight: 998.81,
   },
+
   SodiumAlum: {
+    CAS: '',
     inchiString: '',
     inchiKey: '',
     molecularWeight: 916.56,
   },
+
   Ammonia: {
+    CAS: '7664-41-7',
     inchiString: 'InChI=1S/H3N/h1H3',
     inchiKey: 'QGZKDVFQNNGYKY-UHFFFAOYSA-N',
     molecularWeight: 17.03,
   },
+
   AmmoniumAcetate: {
+    CAS: '631-61-8',
     inchiString: 'InChI=1S/C2H4O2.H3N/c1-2(3)4;/h1H3,(H,3,4);1H3',
     inchiKey: 'USFZMSVCRYTOJT-UHFFFAOYSA-N',
     molecularWeight: 77.08,
   },
+
   AmmoniumAuricyanide: {
+    CAS: '',
     inchiString: '',
     inchiKey: '',
     molecularWeight: 337.09,
   },
+
   AmmoniumBicarbonate: {
+    CAS: '1066-33-7',
     inchiString: 'InChI=1S/CH2O3.H3N/c2-1(3)4;/h(H2,2,3,4);1H3',
     inchiKey: 'ATRRKUHOCOJYRX-UHFFFAOYSA-N',
     molecularWeight: 79.06,
   },
+
   AmmoniumBromide: {
+    CAS: '12124-97-9',
     inchiString: 'InChI=1S/BrH.H3N/h1H;1H3',
     inchiKey: 'SWLVFNYSXGMGBS-UHFFFAOYSA-N',
     molecularWeight: 97.94,
   },
+
   AmmoniumCarbonate: {
+    CAS: '506-87-6',
     inchiString: 'InChI=1S/CH2O3.2H3N.H2O/c2-1(3)4;;;/h(H2,2,3,4);2*1H3;1H2',
-    inchiKey: 'MMVILKIFGQPOLE-UHFFFAOYSA-N',
+    inchiKey: 'PRKQVKDSMLBJBJ-UHFFFAOYSA-N',
```

Comment:
- Older InChI key is correct.
- Ref: https://pubchem.ncbi.nlm.nih.gov/compound/86740279